em/scripts/flower.py

```diff
@@ -167,9 +165,9 @@ def main():
         lnk_label += '_'
         for j in i:
             lnk_label += j.lower()
-    params = CP.read_charmm_FF(param_path)
+    params = CP.read_charmm_FF()
     cmc = md.CenterOfMassCalculator(params)
-    rig = MRM.Molecular_Rigid_Manipulation(param_path)
+    rig = MRM.Molecular_Rigid_Manipulation(params)
```

Comment:
seems to me there was a bug here, it was expecting a params object not a string. Haven't run this though

Reply:
It all looks fine except for this. I will accept the merge because even if it is not a bug it would be easy to patch later.